backend/src/migrations/20230828113344_add_user_details_id_to_talent_badges_table.ts

```diff
@@ -0,0 +1,35 @@
+import { type Knex } from 'knex';
+
+const TableName = {
+    USER_DETAILS: 'user_details',
+    TALENT_BADGES: 'talent_badges',
+};
+
+const ColumnName = {
+    ID: 'id',
+    USER_DETAILS_ID: 'user_details_id',
+};
+
+const RelationRule = {
+    CASCADE: 'CASCADE',
+    SET_NULL: 'SET NULL',
+} as const;
+
+async function up(knex: Knex): Promise<void> {
+    return knex.schema.alterTable(TableName.TALENT_BADGES, (table) => {
```

Comment:
Why we use alterTable if migration name is add_user_details, and previously we didn't has this table ?

Also name migrations like: 
- add_${tableName}_table.ts - to create table
- update_${tableName}_table.ts - to update table